db/migrate/20170626122608_set_public_body_translation_defaults.rb

```diff
@@ -0,0 +1,20 @@
+# -*- encoding : utf-8 -*-
+class SetPublicBodyTranslationDefaults < ActiveRecord::Migration
+  def up
+    change_column :public_body_translations,
+                  :short_name, :text, :null => false, :default => ""
+    change_column :public_body_translations,
+                  :notes, :text, :null => false, :default => ""
+    change_column :public_body_translations,
+                  :publication_scheme, :text, :null => false, :default => ""
+  end
```

Comment:
Presumably creates and saves a translation for each locale when you submit the form. Because we don't have the same defaults and constraints on the `public_body_translations` table we get the mismatch.